tests/_data/recipes/src/integrator-recipes-master/TestIntegratorWirePlugin/1.0.2/installer-manifest.json

```diff
@@ -2,7 +2,8 @@
     "wire-plugin": [
         {
             "target": "\\Pyz\\Zed\\TestIntegratorWirePlugin\\TestIntegratorWirePluginDependencyProvider::getTestPlugins",
-            "source": "\\Spryker\\Zed\\TestIntegratorWirePlugin\\Communication\\Plugin\\TestIntegratorWirePlugin"
+            "source": "\\Spryker\\Zed\\TestIntegratorWirePlugin\\Communication\\Plugin\\TestIntegratorWirePlugin",
+            "index": "\\Spryker\\Zed\\TestIntegratorWirePlugin\\TestIntegratorWirePluginConfig::TEST_INTEGRATOR_WIRE_PLUGIN"
```

Comment:
Does it work as well when you just have a string here?

Reply:
Yes